### ISSUE TITLE
Creating custom metrics with using only numpy library?

### ISSUE THREAD

**Issue by subtleIsTheLord** [keras-team/keras#9640]
I am trying to create a custom metric in keras of form **custom_metric(y_true,y_pred)**. Most of the examples they use **import keras.backend as K** and then all the calculation is done using methods inside K. 
Is it not possible to use just numpy library inside **custom_metric** function to do all the calculation and come up with one scalar output?

**Comment by Dref360** [comment]
It's not recommended since it's gonna be much slower, but you can use https://www.tensorflow.org/api_docs/python/tf/py_func
